styles/component-styles/badge.css

```diff
@@ -0,0 +1,38 @@
+.badge-container{
+    display: flex;
+    justify-content: space-around; 
+    align-items: center;
+    background-color: var(--primary-color);
+    flex-wrap: wrap; 
+}
+
+.badge{
+    background-color: red;
+    height: 22px;
+    width: 22px;
+    display: flex;
+    border-radius: 99rem;
+    justify-content: center;
+    align-content: center;
```

Comment:
you can use a utility class for the same (except the border-radius), since it will be used at many places

Reply:
By utility class, do you want me to use these properties in main-styles.css so that they can be re-used or something else.